ios-device-identifiers.json

```diff
@@ -161,6 +162,12 @@
   "iPad16,4": "iPad Pro 11 inch 5th Gen",
   "iPad16,5": "iPad Pro 12.9 inch 7th Gen",
   "iPad16,6": "iPad Pro 12.9 inch 7th Gen",
+  "iPad15,7": "iPad (A16) (WiFi)",
+  "iPad15,8": "iPad (A16) (WiFi+Cellular)",
+  "iPad15,3": "iPad Air (11-inch) (M3) (WiFi)",
+  "iPad15,4": "iPad Air (11-inch) (M3) (WiFi+Cellular)",
+  "iPad15,5": "iPad Air (13-inch) (M3) (WiFi)",
```

Comment:
@filipejtdias Thanks! I think I'd like to stick to using generations for consistency like adamawolf's gist: https://gist.github.com/adamawolf/3048717 (unless there's evidence for apple's preference for a specific name, but then we should update the lot). If you can fix that would be great, but if not, I'll amend when I merge